src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -62,11 +62,12 @@ class ButtonBuilder implements \IteratorAggregate, FormBuilderInterface
      */
     public function __construct($name, array $options = array())
     {
-        if (empty($name) && 0 != $name) {
+        $name = (string) $name;
```

Comment:
may it it worth to add '@toDO' for in >= PHP7.0 you will able to use 'string' as primitive type and it will be casted to string automatically?